src/main/java/com/google/ehub/utility/QueryUtility.java

```diff
@@ -0,0 +1,61 @@
+package com.google.ehub.utility;
+
+import com.google.appengine.api.datastore.Query.CompositeFilterOperator;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+
+/**
+ * Utility class used to query items in Datastore.
+ */
+public final class QueryUtility {
```

Comment:
Just an FYI but we usually just use the term `Utils` instead of `Utility`

Reply:
OK I might make that change in another PR